braket_features/Getting_Devices_and_Checking_Device_Properties.ipynb

```diff
@@ -45,7 +45,7 @@
   },
```

Comment:
<p>You probably meant to get rid of Aspen-8 here instead.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Oh, I see. Never mind, thanks</p>